server/src/plugins/support.ts

```
@@ -0,0 +1,20 @@
+import fp from 'fastify-plugin'
+
+export interface SupportPluginOptions {
+  // Specify Support plugin options here
+}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)